src/core/models.py

```diff
@@ -129,6 +129,11 @@ def profile_images_upload_path(instance, filename):
                    (u'VG', u'Virgin Islands, British'), (u'VI', u'Virgin Islands, U.S.'), (u'WF', u'Wallis and Futuna'),
                    (u'EH', u'Western Sahara'), (u'YE', u'Yemen'), (u'ZM', u'Zambia'), (u'ZW', u'Zimbabwe')]
 
+FIGURE_MIME_TYPES = [
+    'application/xml',
+    'text/html',
```

Comment:
shouldn't these be `core.files.HTML_MIMETYPES` + `XML_MIMETYPES`

I'd probably change the name to `MIMETYPES_WITH_FIGURES`